python_modules/dagster/dagster/_core/storage/event_log/in_memory.py

```diff
@@ -236,6 +236,27 @@ def _apply_filters(record):
                 and record.timestamp >= event_records_filter.before_timestamp
             ):
                 return False
+
+            if (
```

Comment:
heads up, you might get merge conflicts with https://github.com/dagster-io/dagster/pull/10154, but it should be easy to resolve.